openshift/hac-dev
# Migrate nginx file to caddy so we have better metrics

## Fixes 
https://issues.redhat.com/browse/HAC-2147


## Description
Second attempt of https://github.com/openshift/hac-dev/pull/323

Platform is using caddy config file (we are supporting both nginx and Caddy at the moment) in order to have easier metrics reports. This PR migrates nginx config to Caddy one (I kept the /apps/hac-dev in place as well, for backwards compatibility)

## Type of change



## How to test or reproduce?

You can try it with this Dockerfile
```
FROM quay.io/cloudservices/caddy-ubi:357c825
COPY ./Caddyfile /opt/app-root/src/Caddyfile
COPY ./dist /opt/app-root/src/dist
COPY ./package.json /opt/app-root/src
WORKDIR /opt/app-root/src
CMD ["caddy", "run", "--config", "/opt/app-root/src/Caddyfile"]
```

And running `docker build . -t hac-dev` and then `docker run -it -p 8000:8000 hac-dev` you should be able to access both http://localhost:8000/api/plugins/hac-dev/plugin-manifest.json and http://localhost:8000/beta/api/plugins/hac-dev/plugin-manifest.json
